cmd/hyperkube/kubectl.go

```diff
@@ -17,14 +17,15 @@ limitations under the License.
 package main
 
 import (
-	"os"
-
+	"github.com/docker/docker/pkg/term"
 	"k8s.io/kubernetes/pkg/kubectl/cmd"
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 )
 
 func NewKubectlServer() *Server {
-	cmd := cmd.NewKubectlCommand(cmdutil.NewFactory(nil), os.Stdin, os.Stdout, os.Stderr)
+	// need to use term.StdStreams to get the right IO refs on Windows
```

Comment:
Is this not a problem on all the clients that have to deal with exec?  Why specifically here?